source/Tests/Testing/Integration/IntegrationTestForwardCancellationTokenToInvocationsAnalyzerFixture.cs

```diff
@@ -319,65 +351,50 @@ async Task M()
     }
 
     Task MethodAsync(string a, string b = null, Object c = null) => Task.CompletedTask;
-    Task MethodAsync(string a, CancellationToken ct) => Task.CompletedTask;    
+    Task MethodAsync(string a, CancellationToken ct) => Task.CompletedTask;
     Task MethodAsync(string a, string b, Object c, CancellationToken ct) => Task.CompletedTask;
 }
 ";
-            var result0 = new DiagnosticResult(Descriptors.Oct2008IntegrationTestForwardCancellationTokenToInvocations()).WithLocation(0);
-            var result1 = new DiagnosticResult(Descriptors.Oct2008IntegrationTestForwardCancellationTokenToInvocations()).WithLocation(1);
-            
-            await Verify.VerifyAnalyzerAsync(container.WithTestingTypes(), result0, result1);
-        }
-        
-        [TestCase]
-        public async Task Diagnostic_UnorderedNamedParameters()
```

Comment:
Removed this test. Decided to not support this case.
- Adding support for this **edge** case results in **worst** overall experience during actual testing.